app/components/publications/publication-sidebar.js

```diff
@@ -0,0 +1,269 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import {
+  task,
+  restartableTask,
+  lastValue
+} from 'ember-concurrency-decorators';
+import { timeout } from 'ember-concurrency';
+import { tracked } from '@glimmer/tracking';
+import moment from 'moment';
+import CONFIG from 'frontend-kaleidos/utils/config';
+
+export default class PublicationsPublicationSidebarComponent extends Component {
+  /**
+   * @argument isOpen
+   * @argument onCollapse
+   * @argument onOpen
+   * @argument didChange: should take arguments (modifiedObject, keyName)
+   */
+  @service store;
+  @service intl;
+  @service toaster;
+  @service publicationService;
+
+  // Tracked props.
+  @tracked newNumacNumber = '';
+  @tracked numberIsAlreadyUsed = false;
+  @tracked numberIsRequired = false;
+  @tracked showConfirmWithdraw = false;
+
+  @lastValue('loadRegulationTypes') regulationTypes;
+  @lastValue('loadPublicationStatus') publicationStatus;
+  @tracked publicationModes;
+
+  constructor() {
+    super(...arguments);
+    this.loadRegulationTypes.perform();
+    this.loadPublicationStatus.perform();
+    this.publicationModes = this.store.peekAll('publication-mode').sortBy('position');
+  }
+
+  get publicationFlow() {
```

Comment:
None. It's just an alias. The var is used a lot in the component js though. Spares writing out the `args` all the time. Would that be enough weight to keep it?

Reply:
I see. No strong opinion from my side. It just hides the clear distinction between arguments and 'internal' variables imo. But trying to assign a value to `publicationFlow` will still throw an error since it's a get-function so no risk there.